docs/modules/serialization/pages/serialization.adoc

```diff
@@ -1,32 +1,75 @@
 = Serialization
+:description: To be able to send data over a network between cluster members and/or clients, you have to serialize it into raw bytes. Hazelcast has many serialization methods to choose from, depending on which Hazelcast topology you are using and what you plan on doing with your data.

```

Comment:
This feels more like how Redis does things. You have to convert the object into raw bytes before making a call to Redis. 

In hazelcast, we serialize the data when users makes the call with his object. Only thing user needs to be careful is the related configuration. 
So instead of the following:
`you have to serialize it into raw bytes`
We may say something like:
`Hazelcast will serialize users objects to raw bytes`

I see a similar wording in the rest of the document as well.